Adding default localhost collector node fails because of failing connection to port 5001

Hi,

I'm struggling to get Taranis NG with the recommended Docker set-up running. I'm unable to add the default collector node which is running on port 5001.
![image](https://user-images.githubusercontent.com/199050/139582922-1040d849-b852-485d-b70e-e2bf56f39ea3.png)

The logs of the `taranis-ng_core_1` shows an error (the other containers' logs) don't show anything interesting:
```
[2021-10-31 12:17:17 +0000] [12] [ERROR] Socket error processing request.
...
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=5001): Max retries exceeded with url: /api/v1/collectors (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f21eaf7be90>: Failed to establish a new connection: [Errno 111] Connection refused'))
```
The request's stracktrace is really long but that's the essence.
It seems the core container is unable to connect to the port:
```
$ docker exec -ti taranis-ng_core_1 ash
/app # wget http://localhost:5001/api/v1/collectors
Connecting to localhost:5001 (127.0.0.1:5001)
wget: can't connect to remote host (127.0.0.1): Connection refused
```
But it works outside the core container:
```
$ wget http://localhost:5001/api/v1/collectors
--2021-10-31 12:22:59--  http://localhost:5001/api/v1/collectors
Resolving localhost (localhost)... 127.0.0.1
Connecting to localhost (localhost)|127.0.0.1|:5001... connected.
HTTP request sent, awaiting response... 401 UNAUTHORIZED

Username/Password Authentication Failed.
```

I'm on commit db7292f6e4f7d4732521018c0de2e0f7c155bb59

--- a/docker/README.md
+++ b/docker/README.md
@@ -41,6 +41,9 @@ docker-compose -f docker/docker-compose.yml up --build
 
 **Voila, Taranis NG is up and running. Visit your instance by navigating to [http://127.0.0.1:8080/](http://127.0.0.1:8080/) using your web browser**.
 
+Your Taranis NG instance now needs to be configured.
+Continue [here](../README.md#connecting-to-collectors-presenters-and-publishers).
+
 <hr />
 
 To import the [sample data](../src/core/scripts/sample_data.py) and create basic user accounts, set the environment variable `TARANIS_NG_SAMPLE_DATA` for the core container to `true`, or import sample data using the [management script](#management-script-how-to) (from another terminal):

